app/main.py

```diff
@@ -1,44 +1,70 @@
+import asyncio
 import time
+from typing import Any, Awaitable
 
 from iot.devices import HueLightDevice, SmartSpeakerDevice, SmartToiletDevice
 from iot.message import Message, MessageType
```

Comment:
Ensure that the Message and MessageType imports are correct and that the Message class has 'device_id', 'msg_type', and 'data' attributes, as they are used in the send_msg method.